Route/RouteCollection.php

```diff
@@ -46,16 +46,31 @@ public function __construct($baseCodeRoute, $baseRouteName, $baseRoutePattern, $
     }
 
     /**
-     * @param string $name
-     * @param string $pattern
-     * @param array  $defaults
-     * @param array  $requirements
-     * @param array  $options
+     * Add route
      *
-     * @return \Sonata\AdminBundle\Route\RouteCollection
+     * @param string $name          Name
+     * @param string $pattern       Pattern (will be automatically combined with @see $this->baseRoutePattern and $name
+     * @param array  $defaults      Defaults
+     * @param array  $requirements  Requirements
+     * @param array  $options       Options
+     * @param string $host          Host
+     * @param array  $schemes       Schemes
+     * @param array  $methods       Methods
+     * @param string $condition     Condition
+     *
+     * @return $this
      */
-    public function add($name, $pattern = null, array $defaults = array(), array $requirements = array(), array $options = array())
-    {
+    public function add(
+        $name,
```

Comment:
> like commas after each array element

Talk to Symfony if you don't like it! :-)

For line longer, I think this could be as appreciation. In this case, parameters are described on PHPdoc block so I'm OK for all parameter on one line.